ProjectTemplates/DotNetTemplate/MonoGame.Templates.FSharp/content/MonoGame.Application.DesktopGL.FSharp/Game1.fs

```diff
@@ -0,0 +1,49 @@
+namespace MGNamespace
+
+open Microsoft.Xna.Framework
+open Microsoft.Xna.Framework.Graphics
+open Microsoft.Xna.Framework.Input
+
+type Game1 () as this =
+    inherit Game()
+ 
+    let mutable _graphics : GraphicsDeviceManager = null
+    let mutable _spriteBatch : SpriteBatch = null
```

Comment:
Seems like `= Unchecked.defaultof<SpriteBatch>` is the norm instead of using `null`.
More info at https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/values/null-values.

Reply:
Fixed.